packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/tiles/service-management/service-management.html

```diff
@@ -5,6 +5,7 @@
     <oui-tile-definition
         data-term="{{:: 'ovhManagerPccDashboardServiceManagement_mailingList_term' | translate }}"
         data-description="{{:: $ctrl.currentUser.email }}"
+        data-ng-if="::$ctrl.currentUser.ovhSubsidiary !== 'US'"
```

Comment:
Shouldn't be based on the region over the subsidiary?

Reply:
Maybe it's also the good time to use feature flipping?

Reply:
@marie-j has started configured it for the dedicated application. 

see: https://github.com/ovh/manager/pull/3466/files?file-filters%5B%5D=.html&file-filters%5B%5D=.json#diff-ea0f3ce3d53e95c8861f210b691dac0dR327-R329

**thought:** maybe we can rebase this Pull Request on `fix/hpc-us`.